hermes_core/data/hermes_default_schema.yaml

```diff
@@ -0,0 +1,97 @@
+DOI:
+  derived: false
+  required: true
+  valid_ check: null
+Data_level:
+  derived: false
+  required: true
+  valid_check: null
+Data_product_descriptor:
+  derived: false
+  required: false
+  valid_check: null
+Data_type:
+  derived: get_data_type
+  required: true
+  valid_check: null
+Data_version:
+  derived: false
+  required: true
+  valid_check: null
+Descriptor:
+  derived: false
+  required: true
+  valid_check: null
+Discipline:
+  derived: false
+  required: true
+  valid_check: null
+Generation_date:
+  derived: get_generation_date
```

Comment:
Is this the name of the function that will be run to generate it? Could you add a README.rst that defines how this file should be used and it works please?